### PR TITLE
Changes of data dir

### PR DESCRIPTION
**Pull request by ezraholm50**

### CODE REVIEW

**Comment by enoch85** [comment]
Does this fix the error I posted in IRC? Have you tested this fix against current master?

The data dir is already created during install, and install works. What I want is the error message to dissapear.

**Comment by enoch85** [comment]
Commit https://github.com/enoch85/ownCloud-VM/pull/15 here instead.

**Comment by ezraholm50** [comment]
Thats abracadabra to me sorry, can't figure out how to import the PR or make a new PR in this patch, sorry.

But I assume it does not fix it since its an installation core error. Nevertheless one of the chmod vars was wrong. the commit in owncloud_installation.sh is just to move stuf over if exist in old data dir.

**Comment by enoch85** [comment]
Have you seen this in master?

https://github.com/enoch85/ownCloud-VM/blob/master/beta/owncloud_install.sh#L22
https://github.com/enoch85/ownCloud-VM/blob/master/beta/owncloud_install.sh#L182
https://github.com/enoch85/ownCloud-VM/blob/master/beta/owncloud_install.sh#L174

OCDATA doesnät have to be moved as it's created during install...

**Comment by enoch85** [comment]
Regarding commiting to the same branch, just open that branch and make changes in it.

File: beta/owncloud_install.sh [hunk 1]
```diff
@@ -187,7 +187,7 @@ echo
 sleep 3
 
 # Set new data dir in config.php
-sudo -u www-data php occ config:system:set datadirectory --value="$OCDATA" 
+#sudo -u www-data php occ config:system:set datadirectory --value="$OCDATA"
```
Comment (enoch85) [inline beta/owncloud_install.sh]: @ezraholm50 Remove this completly instead of hashing it out please.

File: beta/owncloud_install.sh [hunk 2]
```diff
@@ -186,6 +186,25 @@ sudo -u www-data php $OCPATH/occ status
...
+ then
+  mv ${OCPATH}/.ocdata ${OCDATA}/.ocdata
+fi
+
+if [ -d ${OCPATH}/data/ocadmin ]
+ then
+  mv ${OCPATH}/data/* ${OCDATA}/
+fi
+
 # Prepare cron.php to be run every 15 minutes
```
Comment (enoch85) [inline beta/owncloud_install.sh]: Remove all this code please. Not needed. @ezraholm50

**Comment by enoch85** [comment]
I've taken what I need from this PR. Thanks for your efforts!

**Pull request closed without merging**